src/asynk/async_runnable.rs

```diff
@@ -37,27 +37,58 @@ impl From<SerdeError> for FangError {
     }
 }
 
+/// Implement this trait to insert your own type of task.
 #[typetag::serde(tag = "type")]
 #[async_trait]
 pub trait AsyncRunnable: Send + Sync {
+    /// This will define what the task do.
     async fn run(&self, client: &mut dyn AsyncQueueable) -> Result<(), FangError>;
 
+    /// This will define what is the task type.
+    /// By default `COMMON_TYPE` constant is the task type
     fn task_type(&self) -> String {
         COMMON_TYPE.to_string()
     }
 
+    /// If set to true, library will handle the uniqueness of the task that implements this trait.
+    /// By default is set to false.
     fn uniq(&self) -> bool {
         false
     }
 
+    /// This will define if a task is periodic or is scheduled once in the time.
```

Comment:
This method defines if a task is periodic or it should be executed once in the future.